package-lock.json

```diff
@@ -3503,7 +3503,7 @@
     },
     "browserify-aes": {
       "version": "1.2.0",
-      "resolved": "https://registry.npmjs.org/browserify-aes/-/browserify-aes-1.2.0.tgz",
+      "resolved": "http://registry.npmjs.org/browserify-aes/-/browserify-aes-1.2.0.tgz",
```

Comment:
Why are these getting turned into `http` links?

Reply:
Who can know! `npm` is the weirdest package manager I've ever used

Reply:
Looks like this can be resolved by clearing your cache before running the `update` command: https://npm.community/t/npm-install-downgrading-resolved-packages-from-https-to-http-registry-in-package-lock-json/1818/7

Reply:
Ok, fixed!